public/app/features/provisioning/Wizard/actions.ts

```diff
@@ -0,0 +1,120 @@
+import { GetRepositoryFilesResponse, GetResourceStatsResponse, RepositoryViewList } from '../api';
+
+export type Target = 'instance' | 'folder';
```

Comment:
```suggestion
export type Target = SyncOptions['target'];
```
Can use the generated type here.

Reply:
nice -- did not realize this is possible